cli/src/add_url.rs

```diff
@@ -3,7 +3,7 @@ use db::urls::{add_url, AddUrlOutcome, PgPool};
 use uuid::Uuid;
 
 use crate::check_auth_status;
-pub async fn append_url_to_db(
+pub async fn append_url(
```

Comment:
changed the name of this function as requested in the previous PR. Let me know if there's an issues 😅